torch/_inductor/compile_fx.py

```diff
@@ -531,6 +536,9 @@ def fx_codegen_and_compile(
                         context.output_strides.append(None)
             compiled_fn = graph.compile_to_fn()
 
+            if _in_aot_compilation:
+                return compiled_fn
```

Comment:
just for my own understanding, why do we need this ?

Reply:
My understanding is, if we're just aot compiling, we do not need to construct the followup `CompiledFXGraph` and we can just return the "compiled_fn" which in this case is just the so_path from AOTCodeCache